srcpkgs/gping/template

```diff
@@ -1,9 +1,11 @@
 # Template file for 'gping'
 pkgname=gping
 version=1.16.1
-revision=1
+revision=2
 build_style=cargo
 make_install_args="--path gping"
+hostmakedepends="pkg-config libgit2-devel"
+makedepends="libgit2-devel"
```

Comment:
if it's just the build script then we need only hostmakedepends

Reply:
Tried that, [didn't work](https://github.com/void-linux/void-packages/actions/runs/8534367101/job/23378632021) for cross. It's not only for build, also in [main]( https://github.com/orf/gping/blob/gping-v1.6.1/gping/src/main.rs#L36).